.github/workflows/build-and-push-services.yml

```diff
@@ -16,8 +16,8 @@ jobs:
           - can_watchdog
           - can_logger
           - socketcand
-          - cand
-          - j1939d
+          # - cand
+          # - j1939d
```

Comment:
Why?

Reply:
Rust emulated compile takes forever and times out (giving failure messages). We can uncomment them as soon as the code is ready to merge.

Reply:
I see. I just figured we could ignore errors for services not in this PR. Need to get the new CI pipeine here now that you can share workflows in GHA.